configure.ac

```diff
@@ -252,7 +252,8 @@ AS_IF([test "x$enable_defaultflags" = "xyes"],
       ADD_COMPILER_FLAG([-Wall])
       ADD_COMPILER_FLAG([-Wextra])
       ADD_COMPILER_FLAG([-Wformat-security])
-      ADD_COMPILER_FLAG([-Werror])
+      AS_IF([test "x$ax_is_release" = "xno"],
```

Comment:
Is this really what we want? With the policy == "dash-version" the $ax_is_release will be "no" for X.Y.Z-dev, but it will be "yes" for RCs (X.Y.Z_rcN) and for the final release, which will case the -Werror flag not be added to them.

Reply:
Hmmm...
What I want is that -Werror is only added to non-releases.
So yes, that's what I want...

Rational being that, if a distro upgrades its compiler, we don't want newly introduce warning to make our releases start failing, due to -Werror.

Reply:
Makes sense. I should read the commit message.